pfdp/css/variables.css

```diff
@@ -16,3 +18,27 @@
   --shadow: 0px 10px 10px rgba(0, 0, 0, 0.1);
   --primary: var(--burgundy);
 }
+
+.burgundy {
```

Comment:
`bg-burgundy`

Reply:
change done

Reply:
moved to utilities with the rest of bg color classes